### PR TITLE
OS and contributors badges added

### PR DESCRIPTION
**Pull request by dhruvhacks**
Adds contribution badge referenced in #26

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 [![Slack](https://img.shields.io/badge/Slack%20channel-%20%20-blue.svg)](http://nirmankarta.herokuapp.com)
 [![PyPI](https://img.shields.io/pypi/v/PyBeacon.svg)](https://pypi.python.org/pypi/PyBeacon)
 [![PyPI](https://img.shields.io/pypi/l/PyBeacon.svg)](https://github.com/nirmankarta/PyBeacon/blob/master/LICENSE)
+[![Contributors](https://img.shields.io/github/contributors/cdnjs/cdnjs.svg)](https://github.com/nirmankarta/PyBeacon/graphs/contributors)
+[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.png?v=103)](https://github.com/nirmankarta/PyBeacon)
```
Comment (PrabhanshuAttri) [inline README.md]: Could you please use shields.io badge? There is a section at the bottom where you can create a badge.
I think there was a miscommunication on my part. OS supported badge was for Operating System supported badge. You can create a badge similar to [this](https://img.shields.io/badge/Operating%20System-Linux-red.svg).

Comment (dhruvhacks) [reply README.md]: Ohh! I am sorry! My mistake.
working on updating this.

**Comment by dhruvhacks** [comment]
@PrabhanshuAttri I have made the required changes! Thank you for your references.
Please review the PR. :slightly_smiling_face:

**Comment by PrabhanshuAttri** [comment]
@dhruvhacks This looks great. Could you please change the base branch from master to dev? Thanks.

**Comment by dhruvhacks** [comment]
@PrabhanshuAttri Sure! Changing target branch from master to dev! :smile:

**Comment by PrabhanshuAttri** [comment]
@dhruvhacks thanks

**Pull request merged**